### PR TITLE
Fix incorrect TS Types for Blocks

### PR DESCRIPTION
**Pull request by ivoberger**
Fixes incorrect types for the Block API Endpoint

### CODE REVIEW

**Comment by aoberoi** [comment]
@ivoberger thanks for quickly patching this for us.

We won't be able to accept your PR until we have our Contributor License Agreement prepared. We certainly appreciate the help and look forward to accepting PRs soon.

I'll leave this open for now, and come back with an update when we're ready.

File: src/api-types.ts [hunk 1]
```diff
@@ -59,56 +59,56 @@ export interface BlockBase {
...
   type: 'heading_1';
-  text: RichText[];
+  heading_1: { text: RichText[] };
   has_children: false;
 }
 
 export interface HeadingTwoBlock extends BlockBase {
   type: 'heading_2';
-  text: RichText[];
+  heading_2: { text: RichText[] };
```
Comment (rss-holmes) [inline src/api-types.ts:74]: Hi👋, over here the key should be heading_3

File: src/api-types.ts [hunk 2]
```diff
@@ -59,56 +59,56 @@ export interface BlockBase {
...
   has_children: false;
 }
 
 export interface HeadingThreeBlock extends BlockBase {
   type: 'heading_3';
-  text: RichText[];
+  heading_2: { text: RichText[] };
```
Comment (rss-holmes) [inline src/api-types.ts:80]: Hi👋, over here the key should be heading_3

Comment (ivoberger) [reply src/api-types.ts]: thx, fixed it

**Comment by blackmad** [comment]
[CLA assistant check](https://cla-assistant.io/makenotion/notion-sdk-js?pullRequest=59) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/makenotion/notion-sdk-js?pullRequest=87) before we can accept your contribution.<br/>

<hr/>

<sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/makenotion/notion-sdk-js?pullRequest=59) it.</sub>

**Comment by ivoberger** [comment]
Signed the CLA and fixed the merge conflicts

**Review by lao9 — APPROVED** [review]
Looks great. Thanks for the fixes.

**Pull request merged**